tap_asana/schemas/subtasks.json

```diff
@@ -342,6 +342,60 @@
         }
       ]
     },
+    "memberships.project": {
```

Comment:
[Querying GET tasks always returns an empty “memberships” object](https://forum.asana.com/t/querying-get-tasks-always-returns-an-empty-memberships-object/90873) Specifying directly memberships.project and memberships.section it works as expected and returns required data